app/javascript/packs/application.js

```diff
@@ -23,6 +24,13 @@ document.addEventListener('DOMContentLoaded', () => {
   Vue.component(VueQrcode.name, VueQrcode);
   Vue.use(VModal);
   Vue.use(VueClipboard);
+  Vue.use(ActionCableVue, {
+    debug: true,
+    debugLevel: 'error',
+    connectionUrl: process.env.RAILS_ENV === 'production' ?
+      ('wss://' + process.env.APPLICATION_HOST + '/cable') : 'ws://localhost:3000/cable',
```

Comment:
Unexpected string concatenation                                     prefer-template<br>'process' is not defined                                            no-undef

Reply:
aquí solo lo pescaría con lo de la concatenación, te está recomendando que uses interpolación, algo como: `wss://${process.env.APPLICATION_HOST}/cable` en vez de suma de strings.

El otro error está bien ignorarlo